### PR TITLE
update guide to mphealth 2.2

### PR DESCRIPTION
**Pull request by DannyS03**
https://github.com/openliberty/guide-microprofile-health/issues/108

### CODE REVIEW

**Comment by gkwan-ibm** [comment]
update pom.xml
- [ ] liberty-maven-plugin 3.2
- [ ] microprofile 3.3
- [ ] junit-jupiter 5.6.2
- [ ] cxf-rt-rs-client 3.3.6
- [ ] cxf-rt-rs-extension-providers 3.3.6
- [ ] commons-lang3 3.10

**Comment by DannyS03** [comment]
> update pom.xml
> 
>     * [ ]  liberty-maven-plugin 3.2
> 
>     * [ ]  microprofile 3.3
> 
>     * [ ]  junit-jupiter 5.6.2
> 
>     * [ ]  cxf-rt-rs-client 3.3.6
> 
>     * [ ]  cxf-rt-rs-extension-providers 3.3.6
> 
>     * [ ]  commons-lang3 3.10

All version already updated.

File: finish/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java [hunk 1]
```diff
@@ -55,12 +55,12 @@ public boolean isHealthy() {
   public HealthCheckResponse call() {
     if (!isHealthy()) {
       return HealthCheckResponse
-          .named(InventoryResource.class.getSimpleName() + "Readiness")
-          .withData("services", "not available").down().build();
+          .builder().name(InventoryResource.class.getSimpleName() + "Readiness")
+          .down("services", "not available").build();
```
Comment (Emily-Jiang) [inline finish/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java:59]: You don't need to call .build() any more.

File: finish/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java [hunk 2]
```diff
@@ -55,12 +55,12 @@ public boolean isHealthy() {
...
     }
     return HealthCheckResponse
         .named(InventoryResource.class.getSimpleName() + "Readiness")
-        .withData("services", "available").up().build();
+        .up("services", "available").build();
```
Comment (Emily-Jiang) [inline finish/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java:63]: You don't need to call .build() any more.

File: finish/src/main/java/io/openliberty/guides/system/SystemReadinessCheck.java [hunk 3]
```diff
@@ -32,16 +32,19 @@ public HealthCheckResponse call() {
...
       // tag::HealthCheckResponse-DOWN[]
-      // tag::HealthCheckResponse-named[]
-      return HealthCheckResponse.named(SystemResource.class.getSimpleName() + "Readiness")
-      // end::HealthCheckResponse-named[]
-                                .withData("default server", "not available").down()
+      // tag::HealthCheckResponse-builder[]
+      return HealthCheckResponse.builder().name(SystemResource.class.getSimpleName() 
+      + "Readiness")
+      // end::HealthCheckResponse-builder[]
+                                .down("default server", "not available")
```
Comment (Emily-Jiang) [inline finish/src/main/java/io/openliberty/guides/system/SystemReadinessCheck.java:39]: You don't need to call .build() any more.

File: finish/src/main/java/io/openliberty/guides/system/SystemReadinessCheck.java [hunk 4]
```diff
@@ -32,16 +32,19 @@ public HealthCheckResponse call() {
...
                                 .build();
       // end::HealthCheckResponse-DOWN[]
     }
     // tag::HealthCheckResponse-UP[]
-    return HealthCheckResponse.named(SystemResource.class.getSimpleName() + "Readiness")
-                              .withData("default server", "available").up().build();
+    return HealthCheckResponse.builder().name(SystemResource.class.getSimpleName() 
+    + "Readiness")
+                              .up("default server", "available")
+                              .build();
```
Comment (Emily-Jiang) [inline finish/src/main/java/io/openliberty/guides/system/SystemReadinessCheck.java:47]: You don't need to call .build() any more.

**Comment by DannyS03** [comment]
- lgdev site updated to review changes. 
- Readiness healthchecks changed to:
```
private static final String readinessCheck = SystemResource.class.getSimpleName() + " Readiness Check";
```

**Comment by DannyS03** [comment]
- tested/verified on **qa**.

**Pull request merged**